src/components/widgets/BookmarkWidget/bookmarkWidget.module.scss

```diff
@@ -39,4 +39,52 @@
 		font-size: 22px;
 		font-weight: 600;
 	}
+
+	.editBookmarkWidget {
+		position: relative;
+		padding: 10px;
+		cursor: pointer;
+
+		.moreOptions {
+			position: absolute;
+			border: 1px solid $widgetTextColor;
+			color: $widgetTextColor;
+			background-color: $widgetBackground;
+			cursor: pointer;
+			border-radius: 16px;
+			width: 140px;
+			height: 111px;
+			top: 10%;
+			// Added to keep the more options modal above the bookmark
+			z-index: 1;
+
+			:first-child {
+				border-bottom: 1px solid $widgetTextColor;
+			}
+
+			.moreOptionButton {
+				display: flex;
+				gap: 5px;
+				padding: 10px;
+
+				.icon {
+					padding: 3px;
+					width: 20px;
+					margin-top: 8px;
+					// Added to remove the border below icon
+					border: none;
+				}
+				.name {
+					padding: 3px;
+					margin: 5px;
+					text-align: left;
+					font-size: 16px;
+				}
+			}
+		}
+	}
+}
+
+.editMenuToggle {
```

Comment:
can this be moved into respective parent? Ignore if this doesn't have a parent element